### ISSUE TITLE
Manual tax for California added to Standard rates when entering a CA shipping address

### ISSUE THREAD

**Issue by gabrielbonifacio** [Automattic/woocommerce-services#2519]
### Describe the bug:

If you have a tax nexus outside California, but enter a California shipping address on the Checkout, the following will happen:

- Taxes are calculated either way on the Checkout, even though you don't need to collect tax on California
- When you go to **WooCommerce > Settings > Tax > Standard rates**, a manual CA rate have been added there.

Even if you delete that tax, it will be re-added as soon as you enter a California address on the Checkout again.

### Steps to reproduce:

1. Make sure your store's base address is on any State besides California. I tested with this base address: `38612 Kalifornsky Beach Rd, Bradenton, FL 34211`.
2. Make sure automated taxes are enabled at **WooCommerce > Tax > Tax options**.
3. Go to **WooCommerce > Tax > Standard rates**, and delete any existing rates for CA.
3. Add a product to the cart and do one of the following:
    - On the Cart page, simulate shipping using a California address. I used this for testing: `San Francisco, CA 94110`.
    - On the Checkout page, enter an address from California.
4. Either of the above will make a tax be added.
5. Go back to **WooCommerce > Tax > Standard rates**, and you'll see a tax rate has been added there.

![https://d.pr/i/7Y5aJr](https://d.pr/i/7Y5aJr+)
_Link to image: https://d.pr/i/7Y5aJr_

![https://d.pr/i/WwcpfL](https://d.pr/i/WwcpfL+)
_Link to image: https://d.pr/i/WwcpfL_

![https://d.pr/i/0U2TZU](https://d.pr/i/0U2TZU+)
_Link to image: https://d.pr/i/0U2TZU_

### Expected behavior:

No tax should be charged, since the tax nexus is outside California and there are no manual rates for that state.

### Actual behavior:

Taxes are added anyway, and also a manual rate are added on WooCommerce's Tax Settings (Standard rates section).

### Isolating the problem:

- [x] I have deactivated other plugins and confirmed this bug occurs when only the extension is active.
- [x] I can reproduce this bug consistently using the steps above.

### Additional details:

Right now I'm aware of two different instances of this issue: 

- 4723156-zen
- 4715683-zen

### WordPress Environment
<details>

```
### WordPress Environment ###

WordPress address (URL): https://gabrielbonifacio.mystagingwebsite.com
Site address (URL): https://gabrielbonifacio.mystagingwebsite.com
WC Version: 6.1.1
REST API Version: ✔ 6.1.1
WC Blocks Version: ✔ 6.5.2
Action Scheduler Version: ✔ 3.4.0
WC Admin Version: ✔ 3.0.3
Log Directory Writable: ✔
WP Version: 5.9
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: ✔

### Server Environment ###

Server Info: nginx
PHP Version: 7.4.27
PHP Post Max Size: 2 GB
PHP Time Limit: 1200
PHP Max Input Vars: 6144
cURL Version: 7.80.0
OpenSSL/1.1.1d

SUHOSIN Installed: –
MySQL Version: 5.5.5-10.4.22-MariaDB-log
Max Upload Size: 2 GB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 6.1.1
WC Database Prefix: wp_
Total Database Size: 42.74MB
Database Data Size: 27.37MB
Database Index Size: 15.37MB
wp_woocommerce_sessions: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.09MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 0.23MB + Index: 0.48MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 0.23MB + Index: 0.19MB + Engine InnoDB
wp_advanced_notifications: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_advanced_notification_triggers: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.05MB + Index: 0.09MB + Engine InnoDB
wp_followup_coupons: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_followup_coupon_logs: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_followup_customers: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_followup_customer_carts: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_followup_customer_notes: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_followup_customer_orders: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_followup_email_excludes: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_followup_email_logs: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_followup_email_orders: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_followup_email_tracking: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_followup_followup_history: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_followup_order_categories: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_followup_order_items: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_followup_subscribers: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_followup_subscribers_to_lists: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_followup_subscriber_lists: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_gla_budget_recommendations: Data: 0.22MB + Index: 0.14MB + Engine InnoDB
wp_gla_merchant_issues: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_gla_shipping_rates: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_gla_shipping_times: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailchimp_carts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailchimp_jobs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_custom_fields: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_dynamic_segment_filters: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_feature_flags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_forms: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_log: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_mapping_to_external_entities: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletters: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_newsletter_links: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_mailpoet_newsletter_option: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_option_fields: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_posts: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_segment: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_newsletter_templates: Data: 2.52MB + Index: 0.00MB + Engine InnoDB
wp_mailpoet_scheduled_tasks: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_scheduled_task_subscribers: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_segments: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_sending_queues: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_settings: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_statistics_clicks: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_mailpoet_statistics_forms: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_statistics_newsletters: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_statistics_opens: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_mailpoet_statistics_unsubscribes: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_mailpoet_statistics_woocommerce_purchases: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_mailpoet_stats_notifications: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_subscribers: Data: 0.02MB + Index: 0.13MB + Engine InnoDB
wp_mailpoet_subscriber_custom_field: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_subscriber_ips: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailpoet_subscriber_segment: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailpoet_user_flags: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_options: Data: 3.05MB + Index: 1.06MB + Engine InnoDB
wp_postmeta: Data: 11.52MB + Index: 8.94MB + Engine InnoDB
wp_posts: Data: 6.52MB + Index: 0.92MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 0.28MB + Index: 0.27MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wcpv_commissions: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wcpv_per_product_shipping_rules: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_deposits_payment_plans: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_deposits_payment_plans_schedule: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_bundle_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_order_composite_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.11MB + Index: 0.17MB + Engine InnoDB
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_warranty_products: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_bundled_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_bundled_items: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_exported_csv_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_gc_activity: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_gc_cards: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_gc_cardsmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_table_rates: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_square_customers: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpml_mails: Data: 0.39MB + Index: 0.00MB + Engine InnoDB
wp_wp_phpmyadmin_extension__errors_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB

### Post Type Counts ###

attachment: 3553
follow_up_email: 3
global_product_addon: 3
mailpoet_page: 1
nav_menu_item: 5
page: 13
po_product_options: 1
product: 852
product-box-item: 3
product_variation: 20
revision: 7
shop_coupon: 5
shop_order: 27
shop_order_refund: 1
sidebar: 1
wc_product_tab: 2

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (3) ###

Jetpack: by Automattic – 10.6-a.3
WooCommerce Shipping & Tax: by WooCommerce – 1.25.20
WooCommerce: by Automattic – 6.1.1

### Inactive Plugins (39) ###

Akismet Anti-Spam: by Automattic – 4.2.2
Checkout Fields Manager for WooCommerce: by QuadLayers – 5.5.1
Extra Product Options for WooCommerce: by actpro – 1.9.5
Follow-Up Emails: by WooCommerce – 4.9.17
Free Gifts for WooCommerce: by FantasticPlugins – 7.4
Google Listings and Ads: by WooCommerce – 1.5.0
Simple Product Box for WooCommerce: by WP1 – 1.0.8
Storefront Powerpack: by WooCommerce – 1.6.1
User Role Editor: by Vladimir Garagulya – 4.61
WooCommerce Account Funds: by Themesquad – 2.6.1
WooCommerce Advanced Notifications: by WooCommerce – 1.2.36
WooCommerce Anti Fraud: by OPMC Australia Pty Ltd – 3.9 (update to version 4.2 is available)
WooCommerce Chained Products: by StoreApps – 2.11.0
WooCommerce Checkout Add-Ons: by SkyVerge – 2.5.6
WooCommerce Checkout Field Editor: by WooCommerce – 1.7.0
WooCommerce Composite Products: by SomewhereWarm – 8.3.2 (update to version 8.3.7 is available)
WooCommerce Customer/Order/Coupon Export: by SkyVerge – 5.3.3
WooCommerce Distance Rate Shipping: by WooCommerce – 1.0.30 (update to version 1.0.31 is available)
WooCommerce EU VAT Number: by WooCommerce – 2.4.1
WooCommerce Gift Cards: by SomewhereWarm – 1.9.1 (update to version 1.10.0 is available)
WooCommerce Google Analytics Integration: by WooCommerce – 1.5.3
WooCommerce Mix and Match Products: by Kathy Darling – 1.11.3 (update to version 1.11.6 is available)
WooCommerce Payments: by Automattic – 3.5.0 (update to version 3.6.0 is available)
WooCommerce Pre-Orders: by WooCommerce – 1.5.30 (update to version 1.6.0 is available)
WooCommerce Print Invoices/Packing Lists: by SkyVerge – 3.11.4
WooCommerce Product Add-ons: by WooCommerce – 4.4.0 (update to version 4.5.0 is available)
WooCommerce Product Bundles: by SomewhereWarm – 6.12.4 (update to version 6.13.3 is available)
WooCommerce Product CSV Import Suite: by WooCommerce – 1.10.46
WooCommerce Product Vendors: by WooCommerce – 2.1.58
WooCommerce Ship to Multiple Addresses: by WooCommerce – 3.6.38 (update to version 3.6.39 is available)
WooCommerce Slack: by WooCommerce – 1.2.8
WooCommerce Smart Coupons: by StoreApps – 4.32.0 (update to version 4.35.0 is available)
WooCommerce Stripe Gateway: by WooCommerce – 6.0.0
WooCommerce Tab Manager: by SkyVerge – 1.14.1
WooCommerce Warranty Requests: by WooCommerce – 1.9.30 (update to version 1.9.31 is available)
WooCommerce Xero Integration: by WooCommerce – 1.7.38 (update to version 1.7.40 is available)
WooSidebars: by WooCommerce – 1.4.5
WP Mail Logging: by Wysija – 1.9.9
WP phpMyAdmin: by Puvox.software – 5.1.1.05

### Dropin Plugins (2) ###

advanced-cache.php: advanced-cache.php
object-cache.php: Memcached

### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: bundle (bundle)
composite (composite)
external (external)
grouped (grouped)
mix and match (mix-and-match)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
exclude_from_catalog (exclude_from_catalog)
exclude_from_search (exclude_from_search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔

### WC Pages ###

Shop base: #5 - /shop/
Cart: #6 - /cart/
Checkout: #7 - /checkout/
My account: #8 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Storefront
Version: 3.8.1 (update to version 3.9.1 is available)
Author URL: https://woocommerce.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Action Scheduler ###

Complete: 973
Oldest: 2021-12-29 00:27:58 +0000
Newest: 2022-01-26 19:43:20 +0000

Pending: 7
Oldest: 2022-01-26 19:44:44 +0000
Newest: 2022-08-21 14:17:39 +0000


### Status report information ###

Generated at: 2022-01-26 16:44:32 -03:00
```

</details>

**Comment by darcie** [comment]
4723579-zen 

I can't find any changes to California taxes for 2022 that would cause the change, and https://developers.taxjar.com/demo/ doesn't show taxes calculating for shipping to California when the store location is elsewhere. 

<img width="1423" alt="Markup 2022-01-26 at 12 56 00" src="https://user-images.githubusercontent.com/318285/151245584-a2cd0ec3-0ca2-438b-9423-b4a90bd1e83b.png">


I can reproduce this in a test store: 

1. I set the store base location to Michigan.
2. With logging on for WooCommerce Shipping & Tax, the log shows `"has_nexus":true` for a California shipping address. 
3. Changing the address on checkout to Florida logged `"has_nexus":false`.

<img width="1244" alt="Markup 2022-01-26 at 13 00 08" src="https://user-images.githubusercontent.com/318285/151246195-f8c6c60c-b5d3-47dc-8c3a-e5ab195a2555.png">

**Comment by namiokuzono** [comment]
4723901-zen 33535695-hc

**Comment by AashikP** [comment]
Another report in 4716415-zen

**Comment by thracefulton** [comment]
Another instance in 31708028-hc. Tax nexus is in FL. The issue began on Monday Jan 24th for them.

**Comment by chrism245** [comment]
Another report 22127360-hc, they didn't have problems before Jan 25th.

**Comment by nicdwilson** [comment]
4727547-zen.
WA based site now collecting taxes for WA and CA. Replicated on a clean staging environment.

**Comment by AashikP** [comment]
More reports in 

4727272-zen
4727288-zen

**Comment by tijosh** [comment]
33911642-hc

**Comment by sebastiank112** [comment]
I'm noticing same issue with AZ > CA orders.

**Comment by shaunkuschel** [comment]
Another report in 4726410-zen (WA-based site, taxes collected for CA orders)

**Comment by frosso** [comment]
It looks like the issue was caused by some API misconfiguration, which is now resovled.
While we investigate this further to prevent additional cases, we recommend going into the WooCommerce settings and manually removing misconfigured values for "US-CA".

This can be done by going into **WooCommerce > Settings > Tax > Standard tax rates**. 
On that page, you can select all the "US CA" tax lines, click "Remove selected rows" and then "save".

![2022-01-28 13 13 57](https://user-images.githubusercontent.com/273592/151607585-b0ae1820-7f9a-4ae8-9103-7c8194e3d554.gif)

In the meantime, we will keep investigating to find a solution that can avoid this manual step.

**Comment by gabrielbonifacio** [comment]
> This can be done by going into **WooCommerce > Settings > Tax > Standard tax rates**. On that page, you can select all the "US CA" tax lines, click "Remove selected 

@frosso In my tests, deleting the rate didn’t work. It would be re-added as soon I as entered a California address again on the Checkout or Cart.

As I workaround, I tried to manually add a tax of 0% for California. That rate got replaced by the plugin as well.

**Comment by jcolombo12350** [comment]
Yes, as soon as someone checks out with a California zipcode, taxes are applied on the checkout page, and a new CA tax rate is added in the 'Standard Tax Rate' page.
This scenario keeps repeating itself.

**Comment by darcie** [comment]
> @frosso In my tests, deleting the rate didn’t work. It would be re-added as soon I as entered a California address again on the Checkout or Cart.
> 
> As I workaround, I tried to manually add a tax of 0% for California. That rate got replaced by the plugin as well.

@gabrielbonifacio That was the case before the adjustment to our API. With the change made today, that should not happen any longer. Are you still seeing that occur?

**Comment by thracefulton** [comment]
In order to prevent future occurrences, please do the following:

1. Navigate to WooCommerce > Settings > Tax  and click `Standard Rates`
2. Select any rows with **State Code** of `CA` (hold the `Shift` key to select more than one at a time)
3. Click the `Remove selected row(s)` button at the bottom of the rate table.

Now that we’ve removed the source of those tax rates being added and applied, any new orders from CA customers should not have tax applied.

**Comment by jcolombo12350** [comment]
Everything is now working correctly.
Thanks for your help. It was very much appreciated...

**Comment by frosso** [comment]
I'm closing this for now.

The API  configuration has been adjusted.
We added a store notice to alert merchants that might have been impacted by the issue. The notice links to the documentation.
We updated the documentation with steps the merchants can follow on how to mitigate the issue: https://woocommerce.com/document/woocommerce-shipping-and-tax/woocommerce-tax/#jan-2022-ca-notice
We added a tool in **WooCommerce > Status > Tools > Delete California tax rates** to help merchants that have been impacted by the issue.

**Comment by namiokuzono** [comment]
I am able to confirm this issue has come back. Re-opening the issue. 
I was looking into an issue reported in 5352039-zen and testing on my staging. Then found out that the CA taxes have been automatically added to Standard Rate tax as described in the original issue reporting.

I have then installed new install of staging site using Jurrasic Ninja, and used COD as payment gateway. The issue is present. Here is the example screenshot of Standard Tax rate that is automatically added after order is placed: https://d.pr/i/fPGZg6

---

I further tested and confirmed that this time CA tax is not added when the Nexus is not CA. (Nexus is CO, and order is placed with CA address = No Standard Rates added)

I have also tested different Nexus (CO) and when CO address is entered for an order, I was able to confirm that CO Standard Tax rate is also entered.

So this is not affecting just CA this time.
